### DISCUSSION TITLE
how to reload on save for toga

### DISCUSSION THREAD

**Discussion by bobwatcherx** [beeware/toga#1782] [Q&A]
is there a way to reload on save .  on toga thanks

**Comment by freakboy3742** [comment]
I have no idea what you're asking for. Reload what? When saving what? It's very hard to give suggestions when you haven't explained what you're trying to do, and what you've already tried.

**Comment by bobwatcherx** [comment]
When I save the file it reloads automatically

**Reply by mhsmith** [reply]
It's still not clear what you mean. Please explain in more detail, and give an example.

**Reply by freakboy3742** [reply]
Oh - do you mean when you're developing an app, and you save your app's code, and the app auto-updates in-place - the sort of thing that is common on modern web development setups?

If that's the case - that's not a feature we currently have.
